.ci/setup_env_ubuntu.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+source "/etc/os-release"
+source "${cidir}/lib.sh"
+get_cc_versions
+
+if grep -q "N" /sys/module/kvm_intel/parameters/nested; then
+	echo "enable Nested Virtualization"
+	sudo modprobe -r kvm_intel
+	sudo modprobe kvm_intel nested=1
+fi
+
+echo "Update apt repositories"
+sudo -E apt update
+
+echo "Install chronic"
+sudo -E apt install -y moreutils
+
+echo "Install kata containers dependencies"
+chronic sudo -E apt install -y libtool automake autotools-dev autoconf bc alien libpixman-1-dev
+
+echo "Install qemu-lite binary"
+"${cidir}/install_qemu_lite.sh" "${qemu_lite_clear_release}" "${qemu_lite_sha}" "$ID"
+
+echo "Install kata-containers image"
+"${cidir}/install_kata_image.sh"
+
+echo "Install CRI-O dependencies for all Ubuntu versions"
+chronic sudo -E apt install -y libglib2.0-dev libseccomp-dev libapparmor-dev libgpgme11-dev
+
+echo "Install bison binary"
+chronic sudo -E apt install -y bison
+
+echo "Install libudev-dev"
+chronic sudo -E apt-get install -y libudev-dev
+
+echo "Install Build Tools"
+sudo -E apt install -y build-essential python pkg-config zlib1g-dev
+
+echo "Install Kata Containers Kernel"
+"${cidir}/install_kata_kernel.sh" "latest"
+
+echo -e "Install CRI-O dependencies available for Ubuntu $VERSION_ID"
+sudo -E apt install -y libdevmapper-dev btrfs-tools util-linux
+
+if [ "$VERSION_ID" == "16.04" ]; then
+	echo "Install os-tree"
+	sudo -E add-apt-repository ppa:alexlarsson/flatpak -y
+	sudo -E apt update
+fi
+
+sudo -E apt install -y libostree-dev
+
+if ! command -v docker > /dev/null; then
+	"${cidir}/../cmd/container-manager/manage_ctr_mgr.sh" docker install
```

Comment:
This tool needs to be imported into kata, but note that a fix for the following is going to be required:

- https://github.com/clearcontainers/tests/issues/881

Reply:
@jodh-intel , I removed the comment about this issue

Reply:
I think this is the last issue then as this won't work currently afaics?